python/semantic_kernel/connectors/ai/ai_request_settings.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict, Optional
+
+from pydantic import Field
+
+from semantic_kernel.sk_pydantic import SKBaseModel
+
+
+class AIRequestSettings(SKBaseModel):
+    service_id: Optional[str] = Field(None, min_length=1)
```

Comment:
Could we add some comments here about what the service_id and extension_data is, how they're used, etc?